lib/python/rosie/browser/util.py

```diff
@@ -149,8 +145,8 @@ def __init__(self, number=0, is_end=False, change_hook=None,
         self.show_controls = show_controls
         self.combo_box = gtk.combo_box_new_text()
         self.char = self.CHARS[is_end]
-        for n in range(max(number, self.MAX_NUM) + 1):
-            self.combo_box.append_text(self.char * n)
+        for i in range(max(number, self.MAX_NUM) + 1):
```

Comment:
Single character variable name.

Reply:
The character *i* is actually allowed as an index variable.

Reply:
That's awfully letterist!